pandas/core/ops.py

```diff
@@ -700,11 +767,44 @@ def wrapper(self, other):
                                  'reverse': None}}
 
 _op_names = list(_op_descriptions.keys())
-for k in _op_names:
-    reverse_op = _op_descriptions[k]['reverse']
-    _op_descriptions[reverse_op] = _op_descriptions[k].copy()
-    _op_descriptions[reverse_op]['reversed'] = True
-    _op_descriptions[reverse_op]['reverse'] = k
+for key in _op_names:
+    reverse_op = _op_descriptions[key]['reverse']
+    if reverse_op is not None:
+        _op_descriptions[reverse_op] = _op_descriptions[key].copy()
+        _op_descriptions[reverse_op]['reversed'] = True
+        _op_descriptions[reverse_op]['reverse'] = key
+
+
+def _make_flex_doc(op_name, typ):
+    """
+    Make the appropriate substitutions for the given operation and class-typ
+    into either _flex_doc_SERIES or _flex_doc_FRAME to return the docstring
+    to attach to a generated method.
+
+    Parameters
+    ----------
+    op_name : str
+    typ : str
+
+    Returns
+    -------
+    doc : str
+    """
+    op_name = op_name.replace('__', '')
+    op_desc = _op_descriptions[op_name]
+
+    if op_desc['reversed']:
+        equiv = 'other ' + op_desc['op'] + ' ' + typ
+    else:
+        equiv = typ + ' ' + op_desc['op'] + ' other'
+
+    if typ == 'series':
+        doc = _flex_doc_SERIES % (op_desc['desc'], op_name, equiv,
```

Comment:
pass these as ``.format()``

Reply:
Will do.  While I'm at it I'm going to bring the Panel docstring over here too and put the docstring templates in a dedicated section of the file.  Other than that nothing changing in the next push (other than addressing comments).